apps/charterafrica/src/lib/ecosystem/airtable/processData.js

```diff
@@ -0,0 +1,194 @@
+import * as Sentry from "@sentry/nextjs";
+
+function getter(data, key) {
+  return key ? data?.[key] : null;
+}
```

Comment:
```suggestion
function getValue(data, key, defaultValue=null) {
  return data?.[key] ?? defaultValue
}
```

But I think typing `getter(fields, name)` or `getValue(fields, name)` is not shorter that just `fields?.[name] ?? null`